src/test/java/seedu/manager/command/MarkEventCommandTest.java

```diff
@@ -18,7 +18,7 @@ public void testSetUp() {
     @Test
     public void execute_eventPresentMarkDone_success() {
```

Comment:
Consider changing these method names to something more intuitive? Also the CI will fail method names with underscores in it

Reply:
Noted, I'll amend them accordingly.